x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/action/TransportXPackInfoActionTests.java

```diff
@@ -93,6 +83,100 @@ protected List<XPackInfoFeatureAction> infoActions() {
         categories.add(XPackInfoRequest.Category.FEATURES);
         request.setCategories(categories);
 
+        XPackInfoResponse response = getResponse(request, client, licenseService, new ArrayList<>(featureSets.keySet()));
+
+        if (request.getCategories().contains(XPackInfoRequest.Category.BUILD)) {
```

Comment:
Not sure why GH renders the diffs this way, these are not new assertions. The existing test has just a little refactoring to allow to better share across the new test scenario.